docs/src/environments/_index.md

```diff
@@ -1,51 +1,86 @@
 If you have a live site, you have at least a production environment.
 You may also have additional environments for development, testing, staging, review, and so on.
+To organize your environments, you can create [hierarchical relationships](#hierarchy).
 
-New environments can be created by branching existing environments using the [command line interface (CLI)](/administration/cli/_index.md),
-or in the [Console](../administration/web/_index.md).
+To create new environments, you can branch existing environments using the [CLI](/administration/cli/_index.md)
+or the [Console](../administration/web/_index.md).
 Each created environment is an exact replica of its parent environment.
-This means new environments have all of the data and services from the parent
-(like databases, network storage, queues, routing).
-
-To organize your environments, you can create [hierarchical relationships](#hierarchy).
+This means new environments inherit all of the data and services from their parent environment.
+This includes databases, network storage, queues, and routing configurations.
 
-An environment is tied to a Git branch and can be created on demand.
-With Bitbucket and GitHub integrations you can even get a "development server" automatically for each and every pull request.
+You can create environments on demand.
+Each environment is tied to a Git branch.
+With [Bitbucket](../integrations/source/bitbucket.md) and [GitHub](../integrations/source/github.md) integrations,
+you can even get a "development server" automatically for every pull request.
 
-You can have branches that aren't tied to a running instance of your application.
-These are called [inactive environments](../other/glossary.md#inactive-environment).
+You can also have branches that aren't tied to a running instance of your application.
+These are called [inactive environments](#environment-status).
 
 ## Default environment
 
-Your default environment comes from your default branch and is a [Production environment](../administration/users.md#environment-types).
+Your default environment comes from your default branch and is a [production environment](../administration/users.md#environment-types).
+Your project must have a default environment,
+but you can [name it as you want](./default-environment.md).
+
+If you subscribed to a production plan, this environment is your **live site**.
+You can give it a [custom domain name](../domains/steps/_index.md) and a [custom TLS certificate](../domains/steps/tls.md).
+
+## Environment status
+
+Your environments can have one of two statuses:
+
+-   [Active](../other/glossary.md#active-environment):
+    A deployed environment with services and data.
+
+-   [Inactive](../other/glossary.md#inactive-environment):
+    An environment that isn't deployed and has no services or data, only code.
+    For example, when you push a local branch created with Git to your project,
+    it creates an inactive environment.
```

Comment:
:lady_beetle: Technically, there's an option to push with the CLI branches created with Git and activate them on the push. I'd also say the `it` is slightly vague.